source/base/Text.ooc

```diff
@@ -32,7 +32,10 @@ Text: cover {
 		this _count = 0
 		this _owner = Owner Unknown
 	}
-	init: func@ ~fromStringLiteral (string: CString, =_count) {
+	init: func@ ~fromStringLiteral (string: CString) {
+		this init(string, strlen(string))
+	}
+	init: func@ ~fromStringLiteralWithCount (string: CString, =_count) {
```

Comment:
To initialize from string partially, I guess.
I included it as i wanted to avoid the use of `strlen()`. If the string is not null-terminated, this function will crash your program if you are lucky. Can even expose your system. I'm not a big fan of this function, that's why I didn't include it and here it is because Simon wanted it.